tests/pages/pages.py

```diff
@@ -943,13 +944,32 @@ def send_to_myself(self, message_type):
 
 
 class ServiceSettingsPage(BasePage):
+    name_input = ClearableInputElement()
+
+    @staticmethod
+    def change_setting_link(setting):
+        return (
+            By.XPATH,
+            f"//a[contains(normalize-space(.),'Change')]/span[contains(normalize-space(.),'{setting}')]/parent::a",
+        )
+
+    def click_change_setting(self, setting):
+        element = self.wait_for_element(self.change_setting_link(setting))
+        element.click()
+
     def check_service_name(self, expected_name):
         name = self.wait_for_element(ServiceSettingsLocators.SERVICE_NAME)
+        print(expected_name)
```

Comment:
Just checking these being left in is intentional? No problem if so

Reply:
good catch - prints aren't captured in the logs unless an error happens - but these aren't needed so I'll remove them